exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -53,6 +53,26 @@ func (p *prwTelemetryOtel) recordTranslatedTimeSeries(ctx context.Context, numTS
 	p.telemetryBuilder.ExporterPrometheusremotewriteTranslatedTimeSeries.Add(ctx, int64(numTS), metric.WithAttributes(p.otelAttrs...))
 }
 
+// RemoteWriteProtoMsg represents the known protobuf message for the remote write
+// 1.0 and 2.0 specs.
+type RemoteWriteProtoMsg string
+
+var (
+	// RemoteWriteProtoMsgV1 represents the `prometheus.WriteRequest` protobuf
+	// message introduced in the https://prometheus.io/docs/specs/remote_write_spec/,
+	// which will eventually be deprecated.
+	//
+	// NOTE: This string is used for both HTTP header values and config value, so don't change
+	// this reference.
+	RemoteWriteProtoMsgV1 RemoteWriteProtoMsg = "prometheus.WriteRequest"
+	// RemoteWriteProtoMsgV2 represents the `io.prometheus.write.v2.Request` protobuf
+	// message introduced in https://prometheus.io/docs/specs/remote_write_spec_2_0/
+	//
+	// NOTE: This string is used for both HTTP header values and config value, so don't change
+	// this reference.
+	RemoteWriteProtoMsgV2 RemoteWriteProtoMsg = "io.prometheus.write.v2.Request"
```

Comment:
Isn't this too much detailed? An OTel collector user that doesn't have the knowledge that we have might not understand what this means, right? What do you think of just using 1.0.0, 2.0.0. Or even just 1.0/2.0?

Reply:
I do like copying what prometheus does, but maybe if we document in the readme?

Reply:
In sync meeting we agreed to follow what prometheus does and go with this.

https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write

Reply:
@dashpole are you okay with this?

Reply:
yes, i'm OK with this.